docs/resources/uberbank_example1.ini

```diff
@@ -134,46 +139,39 @@ approximant = SEOBNRv2_ROM_DoubleSpin_HI
 iterative-match-df-max = 8.0
 cache-waveforms =
 
+mtotal-max = 100
+mratio-min = 1
+mratio-max = 97.989
+
+aligned-spin =
+bh-spin-min = -${workflow|max-bh-spin}
+bh-spin-max = ${workflow|max-bh-spin}
+ns-spin-min = -${workflow|max-ns-spin}
+ns-spin-max = ${workflow|max-ns-spin}
+ns-bh-boundary-mass = ${workflow|nsbh-boundary}
+
 [sbank-bbh]
 ; These options are sent to all sbank "BBH" jobs
 
 ; Mass/spin parameter options
-mass1-min = 5.0
-mass1-max = 100.0
-mass2-min = 5.0
-mass2-max = 100.0
-mtotal-min = 10.0
-mtotal-max = 100.0
-mratio-min = 1.0
-mratio-max = 3.0
-aligned-spin =
-spin1-max = 0.9899
-spin1-min = -0.9899
-spin2-max = 0.9899
-spin2-min = -0.9899
```

Comment:
Just realised that I missed the ns-bh boundary mass in the original version. So the final bank would have populated BNS with large spins as well! That might explain a lot of the bank size increase.